src/api/TodoListApi.ts

```diff
@@ -1,7 +1,19 @@
 import { API_URL } from "@todo/consts";
+import { TodoItem } from "@todo/store";
 
 const getTodoList = async () => {
   return await fetch(`${API_URL}/items`);
 }
 
-export { getTodoList }
+const postTodoListItem = async (newItem: TodoItem) => {
+  await fetch(`${API_URL}`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      'Accept': 'application/json',
```

Comment:
Not sure why you need these headers, json is by default for all requests.

Reply:
I did it because I thought use real Database. That's why included headers at once.